src/FileFormats/FileFormats.jl

```diff
@@ -88,23 +87,18 @@ function Model(
     end
 end
 
-const MATH_OPT_FORMATS = Union{
-    CBF.Model,
-    LP.Model,
-    MOF.Model,
-    MPS.Model,
-    SDPA.Model,
-}
+const MATH_OPT_FORMATS =
+    Union{CBF.Model,LP.Model,MOF.Model,MPS.Model,SDPA.Model}
 
 function MOI.write_to_file(model::MATH_OPT_FORMATS, filename::String)
     compressed_open(filename, "w", AutomaticCompression()) do io
```

Comment:
`return` in `do` blocks is a bit weird, is there a way to avoid them?

Reply:
`do` is a function block, so there should be a `return` here. It clarifies that you can go
```Julia
x = foo() do arg
    return arg + 1
end
```
and `x` will have the value `arg + 1`.